CONTRIBUTING.md

```diff
@@ -17,6 +17,20 @@ and what are the requirements for a Pull Request to be opened against Raiden.
 There are two ways you can contribute to the development. You can either open
 an Issue or if you have programming abilities open a Pull Request.
 
+## Setup development environment
+### Installation
+- Install [pyenv](https://github.com/pyenv/pyenv) and [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv) and create a separate virtual environment for your raiden installation
+- Make sure that [tox](https://tox.readthedocs.io/en/latest/) is installed
+- Install [solidity](https://solidity.readthedocs.io/en/latest/installing-solidity.html) and ensure that ```solc --version``` prints out something
+
+### Dependencies
+- Run ```pip install -e .``` to install the dependencies
```

Comment:
To run the tests you also need the development requirements. These are  in `dev_requirements.txt`.

So the only one thing you need to do is `pip install -r dev_requirements.txt`